src/content/components/tooltip/usage.mdx

```diff
@@ -4,34 +4,73 @@ tabs: ['Code', 'Usage','Style']
 ---
 ## General guidance
```

Comment:
I think you should just kill this section. It serves no real purpose.

Reply:
I agree, this whole page needs work. Part of the usage overhaul!